.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: Run tests
+on: pull_request
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
+      - uses: subosito/flutter-action@v1
+        with:
+          flutter-version: '2.0.0'
```

Comment:
You can remove this 2.0.0. with next line (channel: stable), get the last stable version